docs/welcome.mdx

```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
 ---
```

Comment:
Let's make these changes in a separate PR with input from Klaus who already has a plan for the contents of this page. You can find a draft here: https://www.notion.so/Introduction-38cc979e1cd444cb99a5a19318283c88

Reply:
the changes are fine tho